src/gui/gt_colors.cpp

```diff
@@ -60,6 +60,20 @@ gt::gui::color::base()
     return currentTheme().color(QPalette::Base);
 }
 
+QColor
+gt::gui::color::headingBackground()
+{
+    if (gtApp && gtApp->inDarkMode())
+    {
+        // Todo: how to compute this color from base?
+        return QColor(10, 17, 31);
+    }
+    else
+    {
+        return gt::gui::color::base();
+    }
```

Comment:
We could calculate a color like this but we also need to update the dock widgets and home screen background as well, as these look off. 

Either way, maybe consider naming the color/function "accentColor" or similar? I think this name would allow the function to be used more widely.

Using the approach gives the following result:

![image](https://github.com/user-attachments/assets/1dd6c63f-9893-4442-8c43-bc8a51f4ee0d)

```suggestion
gt::gui::color::mainAccentColor()
{
    bool darken = gtApp && gtApp->inDarkMode();
    return gt::gui::color::main().darker(darken ? 150 : -150);
```

Reply:
I have played around with it, also changing luminescence and hue values. All came near to the desired result, but were 1% off. I did not like the results. It needs to be exactly this color with a slight blue tone... Nothing else ;)